tests/unit/core/components/TestSurveyNotice.js

```diff
@@ -0,0 +1,102 @@
+import querystring from 'querystring';
+
+import config from 'config';
+import * as React from 'react';
+
+import SurveyNotice, { SurveyNoticeBase } from 'core/components/SurveyNotice';
+import { dismissSurvey } from 'core/reducers/survey';
+import Notice from 'ui/components/Notice';
+import { dispatchClientMetadata } from 'tests/unit/amo/helpers';
+import {
+  fakeCookie,
+  fakeI18n,
+  fakeRouterLocation,
+  getFakeConfig,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  const render = ({
+    store = dispatchClientMetadata().store,
+    ...customProps
+  } = {}) => {
+    const props = {
+      _config: getFakeConfig({
+        enableExperienceSurvey: true,
+      }),
+      i18n: fakeI18n(),
+      location: fakeRouterLocation(),
+      store,
+      ...customProps,
+    };
+    return shallowUntilTarget(<SurveyNotice {...props} />, SurveyNoticeBase);
+  };
+
+  it('configures a Notice', () => {
+    const root = render();
+
+    const notice = root.find(Notice);
+    expect(notice).toHaveProp('type', 'generic');
+    expect(notice).toHaveProp('dismissible', true);
```

Comment:
Is it worth also asserting about other properties of `Notice`, such as `actionText` and `id`?

Reply:
I don't think so. Actually, this test was a placeholder and I should probably delete it. I don't think it's worth adding tests for statically configured properties. Once a property dynamically depends on conditional logic, it's worth adding a test.